infrastructure/cdn-in-a-box/cache/Dockerfile

```diff
@@ -60,6 +60,7 @@ RUN dnf -y install epel-release && \
         libmaxminddb            \
         libtool-ltdl            \
         libunwind               \
+        libuuid                 \
```

Comment:
`rockylinux:8`, `centos:7`, and `centos:8` all include the `libuuid` package. Is specifying it needed?

```shell
[zrhoffman@computer ~]$ for image in rockylinux:8 centos:{7,8}; do docker run --rm "$image" rpm -q libuuid; done
libuuid-2.32.1-35.el8.x86_64
libuuid-2.23.2-65.el7.x86_64
libuuid-2.32.1-27.el8.x86_64
```

Reply:
Even if CentOS includes it by default, if ATS needs it, specifying it will make the Dockerfile more portable, easier to switch base images to other things that might not have it

Reply:
Works for me :+1: